src/main/java/chatbot/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,54 @@
+package chatbot.commands;
+
+import chatbot.storage.Storage;
+import chatbot.tasks.Deadline;
+import chatbot.tasks.Task;
+import chatbot.tasks.TaskList;
+import chatbot.ui.Ui;
+
+/**
+ * Represents a command to add a deadline to the task list
+ */
+public class DeadlineCommand extends Command {
+    /**
+     * Description of the deadline to be added
+     */
+    String description;
+
+    /**
+     * Due date for the task
+     */
+    String date;
```

Comment:
Minor issue, but these fields are lacking access modifiers. Maybe they should be set to private?

```suggestion
    /**
     * Description of the deadline to be added
     */
    private String description;

    /**
     * Due date for the task
     */
    private String date;
```